src/loader.rs

```diff
@@ -108,9 +108,33 @@ impl AnalysisLoader for CargoAnalysisLoader {
     }
 }
 
+fn extract_target_triple(sys_root_path: &Path) -> String {
+    // First try to get the triple from the rustc version output,
```

Comment:
Could we also add that we assume target to be our host and that it's also not ideal at the moment, as per [your comment](https://github.com/rust-dev-tools/rls-analysis/pull/160#issuecomment-446804440?